### PR TITLE
(PC-8361) design force update

### PR DESCRIPTION
**Pull request by marionnousvalentin**
Link to JIRA ticket: https://passculture.atlassian.net/browse/PC-8361

## Checklist

I have:

- [x] Made sure the title of my PR follows the convention `[PC-XXX] <summary>`.
- [x] Made sure my feature is working on the relevant real / virtual devices.
- [x] Written **unit tests** for my feature.
- [ ] Added new reusable components to the AppComponents page and communicate to the team on slack
- [x] Added a **screenshot** for UI tickets.
- [ ] Attached a **ticket number** for any added TODO/FIXME \
       (for tech tasks, give the best context about the TODO resolution: what/who/when).

## Screenshots

| \*iOS - [Phone name] | \*Android - [Phone name] | Tablet - [Phone name] | Little - [Phone name] |
| -------------------- | :----------------------: | --------------------: | --------------------: |
|          
image
            |                          |                       |                       |

## Deploy hard

If native code (ios/android) was modified, **after** the PR is merged, on the master branch, upgrade the **app version** (+1 patch):

- if you want an hard deployment of the testing environment, use `yarn trigger:testing:deploy`

### CODE REVIEW

File: src/features/forceUpdate/ForceUpdate.tsx [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+import styled from 'styled-components/native'
+
+import { Background } from 'ui/svg/Background'
+import { Star } from 'ui/svg/icons/Star'
+import { ColorsEnum, getSpacing, Spacer, Typo } from 'ui/theme'
+
+import { ButtonPrimaryWhite } from '../../ui/components/buttons/ButtonPrimaryWhite'
+import { openExternalUrl } from '../navigation/helpers'
+
+const ANDROID_STORE_LINK = 'https://play.google.com/store/apps/details?id:app.passculture'
```
Comment (antoinewg) [inline src/features/forceUpdate/ForceUpdate.tsx:13]:
```suggestion
const ANDROID_STORE_LINK = `https://play.google.com/store/apps/details?id:${ANDROID_APP_ID}`
```

File: src/features/forceUpdate/ForceUpdate.tsx [hunk 2]
```diff
@@ -0,0 +1,77 @@
+import { t } from '@lingui/macro'
+import React from 'react'
+import { Platform } from 'react-native'
+import styled from 'styled-components/native'
+
+import { Background } from 'ui/svg/Background'
+import { Star } from 'ui/svg/icons/Star'
+import { ColorsEnum, getSpacing, Spacer, Typo } from 'ui/theme'
+
+import { ButtonPrimaryWhite } from '../../ui/components/buttons/ButtonPrimaryWhite'
```
Comment (antoinewg) [inline src/features/forceUpdate/ForceUpdate.tsx:10]:
```suggestion
import { ButtonPrimaryWhite } from 'ui/components/buttons/ButtonPrimaryWhite'
```

File: src/features/navigation/RootNavigator/routes.ts [hunk 3]
```diff
@@ -42,6 +42,8 @@ import { LocationFilter } from 'features/search/pages/LocationFilter'
 import { LocationPicker } from 'features/search/pages/LocationPicker'
 import { SearchFilter } from 'features/search/pages/SearchFilter'
 
+import { ForceUpdate } from '../../forceUpdate/ForceUpdate'
```
Comment (antoinewg) [inline src/features/navigation/RootNavigator/routes.ts:45]:
```suggestion
import { ForceUpdate } from 'features/forceUpdate/ForceUpdate'
```

**Review by AnoukHello — COMMENTED** [review]
Je n'ai pas vu la maquette mais est-ce qu'on pourrait élargir le bouton pour qu'on voit "Télécharger la dernière version" en entier ?

File: src/features/forceUpdate/ForceUpdate.tsx [hunk 4]
```diff
@@ -0,0 +1,89 @@
...
+
+import { openExternalUrl } from 'features/navigation/helpers'
+import { env } from 'libs/environment'
+import { ButtonPrimaryWhite } from 'ui/components/buttons/ButtonPrimaryWhite'
+import { Background } from 'ui/svg/Background'
+import { Star } from 'ui/svg/icons/Star'
+import { ColorsEnum, getSpacing, Spacer, Typo } from 'ui/theme'
+
+const ANDROID_STORE_LINK = `https://play.google.com/store/apps/details?id:${env.ANDROID_APP_ID}`
+const IOS_STORE_LINK = 'https://apps.apple.com/us/app/pass-culture/id1557887412'
```
Comment (antoinewg) [inline src/features/forceUpdate/ForceUpdate.tsx:14]: pareil ici

Comment (antoinewg) [reply src/features/forceUpdate/ForceUpdate.tsx:14]: avec 1557887412

**Review by AnoukHello — APPROVED** [review]

**Pull request merged**